web/frontend/src/index.tsx

```diff
@@ -224,38 +224,42 @@ const AppContainer = () => {
     };
 
     async function fetchData() {
+      const response = await fetch(ENDPOINT_PERSONAL_INFO, req);
+      let result;
       try {
-        const res = await fetch(ENDPOINT_PERSONAL_INFO, req);
-
-        if (res.status !== 200) {
-          const txt = await res.text();
-          throw new Error(`unexpected status: ${res.status} - ${txt}`);
+        if (response.status === 200) {
```

Comment:
how do you feel about a switch statement ? 😁

Reply:
I changed it. Looks good 👍